src/Apps/Artwork/Components/ArtistInfo.tsx

```diff
@@ -102,7 +112,9 @@ export class ArtistInfo extends Component<ArtistInfoProps> {
                   <Spacer mb={1} />
                   <ArtistBio
                     bio={this.props.artist}
-                    onReadMoreClicked={this.trackArtistBioReadMoreClick}
+                    onReadMoreClicked={this.trackArtistBioReadMoreClick.bind(
```

Comment:
oh yea totally, i usually do `onClick={() => this.onClick` as well (to avoid `bind(this)` language).